public_html/php/Widar.php

```diff
@@ -44,8 +44,8 @@ public function __construct ( /*string*/ $toolname = '', $cookie_lifetime = null
 			);
 			$this->oa = new MW_OAuth ( WbstackMagnusOauth::getOauthParams(
 				'widar',
-				'/tools/widar',
-			), $cookie_lifetime ) ;
+				'/tools/widar'
```

Comment:
PHP 7 doesn't seem to allow trailing commas in argument lists: https://php.watch/versions/8.0/trailing-comma-parameter-use-list

Reply:
Makes sense, I'm not actually sure if this is relevant but surely we want to bump up the two running instances we have (cradle and quickstatements) to newer php versions if we can. Let's not worry about this now though